code/modules/surgery/organs/organ_internal.dm

```diff
@@ -60,6 +60,17 @@
 
 //Special is for instant replacement like autosurgeons
 /obj/item/organ/proc/Remove(mob/living/carbon/M, special = FALSE)
+	//Stop any reagent metabolizing on organ destruction
+	if(reagents && iscarbon(owner))
+		var/mob/living/carbon/body = owner
+		for(var/chem in reagents.reagent_list)
+			var/datum/reagent/_reagent = chem
```

Comment:
This should probably just be named `reagent`. I personally do underscore names when I'm only using something for a typecast or its unused.

Reply:
ha and i was like that is a good way to do names in type casts